src/core/thread.c

```diff
@@ -615,6 +615,10 @@ struct mCoreThread* mCoreThreadGet(void) {
 struct mCoreThread* mCoreThreadGet(void) {
 	return NULL;
 }
+
+void mCoreThreadMarkCrashed(struct mCoreThread* threadContext) {
```

Comment:
Where is this function used in the minimal core build?

Reply:
right below, in line 632. thread is going to be null so it would not be called. but with optimization disabled the compiler/linker can't know.

Reply:
I'd recommend putting THREADING_DISABLED guards around *that* instead of dummying out here.

Reply:
Sure, done